python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -113,6 +113,16 @@ def test_lock(self):
         self.assertEqual(msg, errors.next()[2])
         self.assertNoMoreErrors(errors)
 
+    def test_file_list(self):
+        base_path='./python/tidy/servo_tidy_tests/test_ignored'
+        file_list = tidy.get_file_list(base_path, only_changed_files=False,
+                        exclude_dirs=[])
```

Comment:
Nit: This should be indented to match the first parenthesis.

(something like)

```python
foo = bar.method(blah=...,
          baz=...)
```